memote/suite/tests/test_biomass.py

```diff
@@ -309,17 +312,19 @@ def test_gam_in_biomass(model, reaction_id):
     """
     ann = test_gam_in_biomass.annotation
     reaction = model.reactions.get_by_id(reaction_id)
-    ann["data"][reaction_id] = biomass.gam_in_biomass(model, reaction)
-    ann["metric"][reaction_id] = 1.0  # Placeholder value.
-    if ann["data"][reaction_id]:
+    test_outcome = biomass.gam_in_biomass(model, reaction)
+    ann["data"][reaction_id] = test_outcome
+    test_outcome = ann["data"][reaction_id]
+    ann["metric"] = 1.0 - float(test_outcome)
```

Comment:
The metric here is not per reaction_id, which may lead to issues in the report. I assume you did this on purpose so that you'd get one comparable metric value per test but did you try generating a snapshot or diff report with this?

Reply:
No, oversight on my part. Thank you for catching this.